Algorithm.Python/readme.md

```diff
@@ -26,7 +26,9 @@ Before we enable python support, follow the [installation instructions](https://
 "algorithm-location": "../../../Algorithm.Python/BasicTemplateAlgorithm.py",
 ```
  3. Rebuild LEAN. This step will ensure that the `Python.Runtime.dll` you set in 2.1 will be used.
+      Note: You should do a complete/clean build to recompile the whole solution from scratch, ignoring anything it's done before.
  4. Run LEAN. You should see the same result of the C# algorithm you tested earlier.
+      Note: If you have multiple Python versions, you can confirm you're using the correct one in Visual Studio with (Ctrl+K,Ctrl+`)
```

Comment:
I wouldn't use hot-keys here because they're not the same for everyone. What's it supposed to bind to? What's the VS feature name, also, this statement doesn't help people who aren't using VS
![image](https://user-images.githubusercontent.com/531416/41626846-b3bd0fc2-73eb-11e8-8829-092a5c3fb7a9.png)

Reply:
For Windows and mac we ask users to download Visual Studio in the general readme. I did not think about the hotkey situation so I will fix that.